src/test/java/com/cosmeticos/payment/SuperpayOneClickClientIntegrationTest.java

```diff
@@ -20,21 +22,212 @@
+
+		Long numeroTransacao = 123L;
+		int idioma = 1;
+		String ip = "1234-1233-1234-2123";
+		String origemTransacao = "";//
+		int parcelas = 1;
+		//String token = "12345"; // Ja eh definido no addcard
+		String campainha = "";
+		String urlRedirecionamentoNaoPago = "oiuoiu.com";//travou?
+		String urlRedirecionamentoPago = "pooiuyu.com";
+		long valor = 50L;
+		long valorDesconto = 0;
+		String bairroEnderecoComprador = "austin";// pensando aki.. deixa eu ver se caso eu coloque na ordem se vai funfar
+		String cepEnderecoComprador = "268723-897";
+		String cidadeEnderecoCompra = "nova iguaçu";
+		String codigoCliente = "0987";
+		//long codigoTipoTelefoneComprador = ;
+		//String complementoEnderecoComprador = "";
+		String dataNascimentoComprador = "09/03/1991";
+		//String emailComprador = "email@email.com"; // Ja eh definido no addcard
+		String enderecoComprador = "rua do peixonalta";
+		String estadoEnderecoComprador = "Rio d Janeiro";
+		String nomeComprador = "Fulano";
+		String numeroEnderecoComprador = "66";
+		String paisComprar = "Brasil";
+		String sexoComprador = "M";
+		String telefoneAdicionalComprador = "98762-0987";
+		String telefoneComprador = "94567-7654";
+		long tipoCliente = 1;
+		long codigoTipoTelefoneAdicionalComprador = 2;
+		String codigoCategoria = "12345-123";
+		String nomeCategoria = "nome";
+		String codigoProduto = "Prdutocodigo";
+		String nomeProduto = "Nomeproduto";
+		int quantidadeProduto = 1;
+		Long valorUnitarioProduto = 50L;
+		String cvv = "948576";
+
+		ResultadoPagamentoWS result = pay(numeroTransacao, idioma, ip, origemTransacao, parcelas,
+
+				token, // Gerado pelo addcard la no inicio deste teste.
+
+				campainha, urlRedirecionamentoNaoPago, urlRedirecionamentoPago, valor, valorDesconto, bairroEnderecoComprador, cepEnderecoComprador, cidadeEnderecoCompra, codigoCliente, dataNascimentoComprador, emailComprador, enderecoComprador, estadoEnderecoComprador, nomeComprador, numeroEnderecoComprador, paisComprar, sexoComprador, telefoneAdicionalComprador, telefoneComprador, tipoCliente, codigoTipoTelefoneAdicionalComprador, codigoCategoria, nomeCategoria, codigoProduto, nomeProduto, quantidadeProduto, valorUnitarioProduto, cvv);
+
+		assertThat(result.getStatusTransacao())
+				.isIn(1, 2, 5); // 5=Transacao em Andamento. Acho q tbm vale como OK.
+
+
+		}
+
+	@Test
+	public void testUpdateCard() {
+
+		String dataValidadeCartao = "01/10";
+		String emailComprador = "deivison.2@gmail.com";
+		Long formaPagamento = 1L;
+		String nomeTitularCartaoCredito = "deivison";
+		String numeroCartaoCredito = "1234567899990";
+
+		String token = addCard(dataValidadeCartao, emailComprador, formaPagamento, nomeTitularCartaoCredito, numeroCartaoCredito);
+
+		String result = (oneClickClient.updateCard(
+				dataValidadeCartao,
+				emailComprador,
+				formaPagamento,
+				nomeTitularCartaoCredito,
+				numeroCartaoCredito,
+				token));
+
+		System.out.println(result);
+
+		assertThat(result);
```

Comment:
Este assertThat esta imcompleto.. Vc sempre precisa dar "assert that something...". Neste caso faltou o "something", que seria .isEqualsTo("???") ou isNotNull(), por exemplo.